src/components/core/ApplicationNavigationBar.tsx

```diff
@@ -10,18 +10,20 @@ interface ApplicationNavigationBarProps {
 export default function ApplicationNavigationBar({ opened }: ApplicationNavigationBarProps) {
 	return (
 		<AppShell.Navbar p="md" hidden={!opened}>
-			<Stack className="h-full" gap="xs">
-				{navigationItems.map(({ title, href, subitems }) => (
-					<RecursiveNavLink key={title} item={{ title, href, subitems }} />
-				))}
+			<RemoveScroll enabled={opened}>
```

Comment:
I couldn't find any documents from mantine, but essentially everything wrapped inside this component will be scrollable, the rest would not?

https://github.com/theKashey/react-remove-scroll

Reply:
Yeah